### DISCUSSION TITLE
Allow spend of pending cooperative closed channel funds

### DISCUSSION THREAD

**Discussion by trevlt** [Ride-The-Lightning/RTL#1378] [Q&A]
**Is your feature request related to a problem? Please describe.**
I've made a couple of cooperative closed channel requests. Unfortunately, LND node negotiated low closing channel fees, and those funds are now unconfirmed. I would like to spend those funds with new transaction to my hardware wallet and thus speed up unconfirmed channel close transactions. I couldn't do it through RTL. Or maybe it is a LND issue?

**Describe the solution you'd like**
Allow spend of pending cooperative closed channel funds

**Comment by saubyk** [comment]
Hi @trevlt are you able to view these unconfirmed transactions in the mempool? If yes, you should be able to fee bump via LND cli (assuming you're comfortable with running commands on a terminal)

**Comment by trevlt** [comment]
Is RLT app made for people so that they don't have to dig through mempool or deal with cli? If yes, then such a feature should be available from the app.

**Comment by saubyk** [comment]
thanks for the feedback. I was trying to be helpful, given the situation you're in. If these options are unworkable for you, you can wait and hope the mempool fee rate gets down and your closing transaction gets picked up.

**Comment by saubyk** [comment]
Moving this issue to a discussion thread, as no development action is expected here
